load_sample.py

```diff
@@ -44,9 +48,10 @@ def _load_sample_units(action_plan_id: str, collection_exercise_id: str, sample_
                                                      action_plan_id=action_plan_id),
                                    content_type='application/json')
 
-            sample_unit = {
-                f'sampleunit:{sample_unit_id}': _create_sample_unit_json(sample_unit_id, sample_row)}
-            sample_units.update(sample_unit)
+            if store_loaded_sample_units:
+                sample_unit = {
+                    f'sampleunit:{sample_unit_id}': _create_sample_unit_json(sample_unit_id, sample_row)}
```

Comment:
you know this isn't JSON, right? I don't really understand why (or how) it works like this.